.buildkite/integration.pipeline.yml

```diff
@@ -4,28 +4,84 @@ env:
   DOCKER_REGISTRY: "docker.elastic.co"
   VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
 
+  # Packaging
+  SNAPSHOT: true
```

Comment:
I'd recommend returning those variables to the packaging script to reduce the scope